sw/device/lib/testing/test_rom/test_rom_start.S

```diff
@@ -139,9 +139,9 @@ _start:
   .globl _start
 
   // Check if AST initialization should be skipped.
-  li   t0, (TOP_EARLGREY_OTP_CTRL_CORE_BASE_ADDR + \
-            OTP_CTRL_SW_CFG_WINDOW_REG_OFFSET + \
-            OTP_CTRL_PARAM_CREATOR_SW_CFG_AST_INIT_EN_OFFSET)
+  li   a0, (TOP_EARLGREY_OTP_CTRL_CORE_BASE_ADDR + \
```

Comment:
@timothytrippel 
updated for the missing load word. Maybe we'll need to propagate this back to `mask_rom`?

Reply:
Good point, filed an issue #11117 to track. I noted that we should probably do one better here and share this code between both ROMs so we only have to update in one place. Once I get confirmation on the solution to the issue, I can fix it.